core/templates/pages/story-editor-page/chapter-editor/chapter-editor-tab.component.spec.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for chapter editor tab controller.
+ */
+
+// TODO(#7222): Remove the following block of unnecessary imports once
+// the code corresponding to the spec is upgraded to Angular 8.
+import { UpgradedServices } from 'services/UpgradedServices';
+// ^^^ This block is to be removed.
+
+describe('Chapter Editor tab', function() {
+  var $scope = null;
+  var ctrl = null;
+  var MockStoryEditorNavigationService = null;
+  var StoryEditorNavigationService = null;
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
```

Comment:
Instead of doing this, can you inject the required dependencies using TestBed (read more about it [here](https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#when-should-the-upgraded-services-be-imported-in-the-test-file))?

Reply:
Done.